notebook/services/kernels/kernelmanager.py

```diff
@@ -177,6 +177,7 @@ async def start_kernel(self, kernel_id=None, path=None, **kwargs):
             self._kernel_connections[kernel_id] = 0
             self.start_watching_activity(kernel_id)
             self.log.info("Kernel started: %s" % kernel_id)
+            self.log.info("Kernel type: %s" % self._kernels[kernel_id].kernel_name)
```

Comment:
Rather than introduce another INFO entry that might result in ambiguities when starting kernels simultaneously, would you be okay with extending `Kernel started: <id>` with a comma-separated "type" clause: `Kernel started: <id>, type: <type>`?

Reply:
Hmm - wondering if `name:` might be more indicative and closer to what we use in the code?

Reply:
Thanks Kevin, great points, my first intention was not to change any existing log, but updated the code to address your comments.